docker-compose.yml

```diff
@@ -16,8 +15,7 @@ services:
       - DATA_DIR=${DATA_DIR-}
```

Comment:
I think with the latest persistence changes, `DATA_DIR` was superseded with `PERSISTENCE`. We might want to add that here as well? /cc @giograno